From 36789fd440fc7251a1fd851acb63cc3f3bb33cce Mon Sep 17 00:00:00 2001
From: Mao Minkai <maominkai@wxiat.com>
Date: Tue, 6 Sep 2022 16:39:04 +0800
Subject: [PATCH] sw64: fix assembly style
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 36789fd440fc7251a1fd851acb63cc3f3bb33cce
Modified-by-SEL: No


Sunway inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I5XTGY

--------------------------------

Signed-off-by: Mao Minkai <maominkai@wxiat.com>
Reviewed-by: He Sheng <hesheng@wxiat.com>
Signed-off-by: Gu Zitao <guzitao@wxiat.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/sw_64/kernel/entry-ftrace.S | 2 +-
 arch/sw_64/kernel/entry.S        | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/sw_64/kernel/entry-ftrace.S b/arch/sw_64/kernel/entry-ftrace.S
index 3f88a9fe2e3e..223dd5fc0808 100644
--- a/arch/sw_64/kernel/entry-ftrace.S
+++ b/arch/sw_64/kernel/entry-ftrace.S
@@ -79,7 +79,7 @@ ftrace_graph_call:				/* ftrace_graph_caller(); */
 						/* "br ftrace_graph_caller" */
 #endif
 	mcount_end
-	ret $31, ($28), 1
+	ret	$31, ($28), 1
 	.end ftrace_caller
 #else /* !CONFIG_DYNAMIC_FTRACE */
 
diff --git a/arch/sw_64/kernel/entry.S b/arch/sw_64/kernel/entry.S
index 67bafd4a930a..cfb8e7c6d7fb 100644
--- a/arch/sw_64/kernel/entry.S
+++ b/arch/sw_64/kernel/entry.S
@@ -403,7 +403,7 @@ __switch_to:
 	stl	$15, TASK_THREAD_S6($16)
 	/* Restore context from next->thread */
 	ldl	$26, TASK_THREAD_RA($17)
-	ldl     $30, TASK_THREAD_SP($17)
+	ldl	$30, TASK_THREAD_SP($17)
 	ldl	$9, TASK_THREAD_S0($17)
 	ldl	$10, TASK_THREAD_S1($17)
 	ldl	$11, TASK_THREAD_S2($17)
-- 
2.33.0

